From fcfa4aaeb7f333c01f28890170d831c1649dbb5b Mon Sep 17 00:00:00 2001
From: Danny Nold <dannynold@freescale.com>
Date: Mon, 16 May 2011 17:02:00 -0500
Subject: [PATCH 0169/2463] ENGR00143486 - MSL: MAX17135 init_data incorrectly
 declared __initdata

- Removed __initdata from regulator_init_data structure declaration.  This
protects the MAX17135 regulator from having its structure overwritten, which
was causing a bug when writing large chunks of memory.

Signed-off-by: Danny Nold <dannynold@freescale.com>
---
 arch/arm/mach-mx5/board-mx50_rdp.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx5/board-mx50_rdp.c b/arch/arm/mach-mx5/board-mx50_rdp.c
index 8a0acbc..bbe9687 100755
--- a/arch/arm/mach-mx5/board-mx50_rdp.c
+++ b/arch/arm/mach-mx5/board-mx50_rdp.c
@@ -250,7 +250,7 @@ static struct regulator_consumer_supply vcom_consumers[] = {
 	},
 };
 
-static struct regulator_init_data max17135_init_data[] __initdata = {
+static struct regulator_init_data max17135_init_data[] = {
 	{
 		.constraints = {
 			.name = "DISPLAY",
-- 
1.7.7.4

